engine/installation/linux/oracle.md

```diff
@@ -187,9 +225,24 @@ a new file each time you want to upgrade Docker.
     $ sudo yum install /path/to/package.rpm
     ```
 
-    The Docker daemon starts automatically.
+    The Docker daemon does not start automatically.
```

Comment:
Step 6 appears to be a duplication of information from step 4. Perhaps it should be split out into its own section on "Enabling and Starting the Docker daemon" which covers both install options.